### PR TITLE
Feat/76 responsive update for the pages of services

### PR DESCRIPTION
**Pull request by Iris99999**
### Descriptions

- Link to UI design (if possible):

### Provide screenshots or videos that the function is working.

https://user-images.githubusercontent.com/49296035/229463847-a8c236fb-7514-4906-9a56-af3d57a2bdaa.mp4

### Provide screenshots of no warning, or error in the console.
image
image
image

- [x] All unit tests passing and coverage of testing is more than 90% unless special needs.
- [x] No merge conflict with the `main` branch, make sure your code is up to date with `main`.
- [x] Check PR title is aligned with your ticket.
- [x] Check your app will run without crashes.
- [x] Check if there are hard-coded values.
- [x] No `//eslint-disable-next-line no-unused-vars`, unless otherwise specified.
- [x] Check you are using `camelCase && PascalCase` correctly.
- [x] Remove unused comments.

### CODE REVIEW

File: components/ServicePage/ServicesItem.tsx [hunk 1]
```diff
@@ -32,35 +33,49 @@ const ServiceItem = ({
...
-          className={styles.service_left_logo}
-          src={icon_path}
-          alt={icon_alt}
-        />
-        <h2 className={`${fontSyne.className} global__uppercase-heading-h2`}>{service_name}</h2>
-
-        <div>
-          <li
-            className={`${fontRobotoMono.className} global__text-meta ${styles.services_left__link}`}
+        <section className={styles.service_item__left_title}>
```
Comment (Double-Twelve) [inline components/ServicePage/ServicesItem.tsx:41]: doesn't find the `service_item__left_title` in `styles`

**Comment by Double-Twelve** [comment]
The following two parts may exceed the scope of the page

image
image

**Comment by Iris99999** [comment]
image
thanks， now when the screen size is less than 390, it will be displayed like the above picture

**Review by Double-Twelve — APPROVED** [review]
Great work!

**Review by jiayuu-u — APPROVED** [review]
Looks all good

**Pull request merged**